### PR TITLE
OSDOCS-5976: MicroShift storage section tweaks

### PR DESCRIPTION
**Pull request by ShaunaDiaz**
Version(s):
4.13, 4.14+

Issue:
https://issues.redhat.com/browse/OSDOCS-5976

Link to docs preview:
https://63439--docspreview.netlify.app/microshift/latest/microshift_storage/generic-ephemeral-volumes-microshift#generic-ephemeral-vols-overview_generic-ephemeral-volumes-microshift
https://63439--docspreview.netlify.app/microshift/latest/microshift_storage/generic-ephemeral-volumes-microshift#generic-ephemeral-vols-procedure_generic-ephemeral-volumes-microshift
https://63439--docspreview.netlify.app/microshift/latest/microshift_storage/understanding-persistent-storage-microshift#pv-phase_understanding-persistent-storage-microshift

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by ShaunaDiaz** [comment]
@copejon and @radeore PTAL and let me know if looks good to you? Thanks!

File: modules/storage-persistent-storage-pv.adoc [hunk 1]
```diff
@@ -238,7 +238,7 @@ cluster.
 
 |===
 
-You can view the name of the PVC bound to the PV by running:
+You can view the name of the PVC when it is bound to the PV by running the following command:
```
Comment (dhellmann) [inline modules/storage-persistent-storage-pv.adoc:241]:
```suggestion
You can view the name of the PVC that is bound to the PV by running the following command:
```

You can get the PVC name other ways. This was about getting the PVC associated with a PV.

Comment (radeore) [reply modules/storage-persistent-storage-pv.adoc]: @ShaunaDiaz I'm OK with this change,  just suggested to update the command from "oc get pv <pv-claim>" to "oc get pv <pv-name>"

**Review by dhellmann — COMMENTED** [review]
/lgtm

**Comment by sabrinajess** [comment]
LGTM. Tagging @ousleyp to confirm review.

**Comment by ousleyp** [comment]
Looks good! :)

**Pull request merged**